src/applications/vaos/tests/e2e/vaos-cypress-helpers.js

```diff
@@ -232,6 +235,14 @@ function mockFeatureToggles() {
             name: 'vaOnlineSchedulingExpressCareNew',
             value: true,
           },
+          {
+            name: 'vaOnlineSchedulingFlatFacilityPage',
+            value: facilityPageV2Enabled,
+          },
+          {
+            name: `cerner_override_668`,
+            value: false,
+          },
```

Comment:
I discovered that in addition to mocking the user facilities with `cy.login()`, we also need to pass this feature toggle, otherwise it would set `isCerner: true` for facility 668. See `src/platform/user/selectors.js` line 29.